pkgs/tools/security/shhgit/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "shhgit";
```

```suggestion
  pname = "shhgit-unstable";
```
Since you're grabbing a revision.

---

The situation around this is totally unclear right now so I am just ignoring this.

---

What do you mean by unclear:

Excerpt from https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

---

There is no mention of pname and version and changing the pname of all packages will break at least 10 to 30 percent. Please refer to multiple issues and PRs about this. Until then pname is moved to version in all PRs I review.